ang/test/civicase/case/actions/services/email-managers-case-action.service.spec.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env jasmine */
+
+(function (_, $) {
+  describe('EmailManagersCaseAction', function () {
+    var EmailManagersCaseAction, CasesMockData, caseObj, actualCRMAlert;
+
+    beforeEach(module('civicase', 'civicase.data'));
+
+    beforeEach(inject(function (_EmailManagersCaseAction_, _CasesData_) {
+      EmailManagersCaseAction = _EmailManagersCaseAction_;
+      CasesMockData = _CasesData_;
+    }));
+
+    beforeEach(function () {
+      caseObj = CasesMockData.get().values[0];
+      actualCRMAlert = CRM.alert;
+      CRM.alert = jasmine.createSpy('CRMAlert');
+    });
+
+    afterEach(function () {
+      CRM.alert = actualCRMAlert;
+    });
+
+    describe('doAction()', function () {
+      describe('when no case manager assigned', function () {
+        beforeEach(function () {
+          delete caseObj.manager;
+
+          EmailManagersCaseAction.doAction(caseObj, 'email', jasmine.any(Function));
+        });
+
+        it('calls CRM.alert and return', function () {
```

Comment:
`it` statements should be as much non technical as possible. so `it('shows an error message')`